test/lib/completions/alias.exp

```diff
@@ -30,8 +30,8 @@ set cmd "alias foo"
 append cmd \002\002\002; # \002 = ^B = Move cursor left in bash emacs mode
 send "$cmd\t"
 expect {
-    -re "^alias foo\r\nbar +foo *\r\n/@alias foo$" { pass "$test" }
-    -re "^alias foo=$" { fail "$test: Wrong cursor position" }
+    -re "^alias foo(\b\b\b)?\r\nbar +foo *\r\n/@alias foo(\b\b\b)?$" { pass "$test" }
+    -re "^alias (foo\b\b\b)?foo=(foo\b\b\b)?$" { fail "$test: Wrong cursor position" }
```

Comment:
Thanks a lot for your work on figuring this out. I opted to revert my earlier commit related to the `\b`'s and set `TERM=dumb` instead of `dummy`; that seems to be the original intent anyway.